tests/system-tests/rdscore/tests/00_validate_top_level.go

```diff
@@ -177,6 +177,38 @@ var _ = Describe(
 				Label("egressip", "egressip-ipv6", "egressip-failover"), reportxml.ID("78283"),
 				rdscorecommon.VerifyEgressIPFailOverIPv6)
 
+			It("Verifies pod-level bonded workloads on the same node and same PF",
+				Label("pod-level-bond", "pod-level-bond-same-node"), reportxml.ID("80958"),
+				rdscorecommon.VerifyPodLevelBondWorkloadsOnSameNodeSamePF)
+
+			It("Verifies pod-level bonded workloads on the same node and different PFs",
+				Label("pod-level-bond", "pod-level-bond-same-node"), reportxml.ID("77927"),
+				rdscorecommon.VerifyPodLevelBondWorkloadsOnSameNodeDifferentPFs)
+
+			It("Verifies pod-level bonded workloads on the different nodes and same PF",
+				Label("pod-level-bond", "pod-level-bond-diff-node"), reportxml.ID("78150"),
+				rdscorecommon.VerifyPodLevelBondWorkloadsOnDifferentNodesSamePF)
+
+			It("Verifies pod-level bonded workloads on the different nodes and different PFs",
+				Label("pod-level-bond", "pod-level-bond-diff-node"), reportxml.ID("78295"),
+				rdscorecommon.VerifyPodLevelBondWorkloadsOnDifferentNodesDifferentPFs)
+
+			It("Verifies pod-level bonded workloads during and after bond active interface fail-over",
+				Label("pod-level-bond", "pod-level-bond-fail-over"), reportxml.ID("79329"),
+				rdscorecommon.VerifyPodLevelBondWorkloadsAfterVFFailOver)
+
+			It("Verifies pod-level bonded workloads after pod bonded interface recovering after failure",
+				Label("pod-level-bond", "pod-level-bond-failure"), reportxml.ID("80489"),
+				rdscorecommon.VerifyPodLevelBondWorkloadsAfterBondInterfaceFailure)
+
+			It("Verifies pod-level bonded workloads after bond interface recovering after both VFs failure",
+				Label("pod-level-bond", "pod-level-bond-failure"), reportxml.ID("80696"),
+				rdscorecommon.VerifyPodLevelBondWorkloadsAfterBothVFsFailure)
+
+			It("Verifies pod-level bonded workloads after pod crashing",
```

Comment:
I suggest adding `MustPassRepeatedly(3)` to stress SR-IOV operator a bit
```suggestion
			It("Verifies pod-level bonded workloads after pod crashing",
			    MustPassRepeatedly(3),
```